### ISSUE TITLE
still possible to get nil errors due to timing issues

### ISSUE THREAD

**Issue by ghost** [relevance/log_buddy#2]
```
You have a nil object when you didn't expect it!
You might have expected an instance of Array.
The error occurred while evaluating nil.split
/Users/rsanheim/.gem/ruby/1.8/gems/relevance-log_buddy-0.4.9/lib/log_buddy/utils.rb:25:in `parse_args&rsquo;
/Users/rsanheim/.gem/ruby/1.8/gems/relevance-log_buddy-0.4.9/lib/log_buddy/mixin.rb:18:in `d&rsquo;
```

**Comment by ghost** [comment]
**still possible to get nil errors due to timing issues**

```
1) UsersController - requires login on signup
    Failure/Error: Unable to find matching line from backtrace
    You have a nil object when you didn&rsquo;t expect it!
    You might have expected an instance of Array.
    The error occurred while evaluating nil.split
    # /Users/rsanheim/.gem/ruby/1.8/gems/relevance-log_buddy-    0.4.10/lib/log_buddy/utils.rb:25:in `parse_args&rsquo;
    # /Users/rsanheim/.gem/ruby/1.8/gems/relevance-log_buddy-0.4.10/lib/log_buddy/mixin.rb:18:in `d&rsquo;
    # examples/controllers/users_controller_example.rb:15
    # examples/controllers/users_controller_example.rb:13
    # examples/controllers/users_controller_example.rb:51
```

**Comment by rsanheim** [comment]
Fix issue where logged line goes away; closed by 7e9e155eddd248ea85a9ecf3a77320f7356c579c
